### ISSUE TITLE
update in documentation required

### ISSUE THREAD

**Issue by cAbhi15** [astrolabsoftware/fink-broker#137]
**Describe the issue**
When setting up fink-broker from scratch following the [documentation](https://fink-broker.readthedocs.io/en/latest/#installation) there are two issues one might face when running `fink_test`:
1. run time error in `simulate_stream.py` (division by zero) since no avro files are present initially after cloning the repo and `len(files)` returns zero
2. the coverage report isn't generated

**Action item**
For the 1 above, either the doc can be updated instructing the user to run the `datasim/download_ztf_alert_data.sh` script before running `fink_test` or the script can be automatically run from within the `simulate_stream.py` using a [subprocess](https://docs.python.org/3/library/subprocess.html#module-subprocess)  

For the 2 above, executing 
`echo "spark.python.daemon.module coverage_daemon" > ${SPARK_HOME}/conf/spark-defaults.conf` 
and updating the `~/.bash_profile` with the following lines resolves the issue :
```
export SPARKLIB=${SPARK_HOME}/python:${SPARK_HOME}/python/lib/py4j-0.10.7-src.zip
export PYTHONPATH="${SPARKLIB}:${FINK_HOME}:${FINK_HOME}/python:$PYTHONPATH"
export PATH="${SPARK_HOME}/bin:${SPARK_HOME}/sbin:${PATH}"
```

Hence either the doc has to be updated including the above steps or changes need to be made to take care of the above automatically

@JulienPeloton How should I proceed with fixing this?

**Comment by JulienPeloton** [comment]
thanks @cAbhi15 for pointing out this missing information!

It can be found in the travis script - and just needs to be reported in the installation procedure. Could you update the Installation section: 

https://github.com/astrolabsoftware/fink-broker/blob/274f89a5446bf48beab9106b4662fdee181a37fa/docs/index.md#L30-L54

with what you wrote above?

**Comment by cAbhi15** [comment]
Sure @JulienPeloton 
I will update the installation procedure.

**Comment by JulienPeloton** [comment]
Fixed in #138
